al_aws_collector.js

```diff
@@ -145,6 +146,10 @@ class AlAwsCollector {
     get application_id () {
         return this._applicationId;
     };
+
+    get awsAccountId () {
```

Comment:
let's stick to one naming convention for props which a class.
please use `aws_account_id`, like `application_id` above.